contents/docs/cdp/batch-exports/bigquery.md

```diff
@@ -0,0 +1,97 @@
+
+Navigate to IAM and click on Grant Access to arrive at this screen:
+
+![Project grant](../../images/docs/batch-exports/bigquery/project-grant.png)
+
+In the screenshot above, we have used a custom role named `Testing PostHog BatchExports` with the permissions listed in the previous step.
+
+6. Create a dataset within a BigQuery project (ours is called `BatchExports`, but any name will do).
+7. Follow the instructions on [granting access to a dataset in BigQuery](https://cloud.google.com/bigquery/docs/control-access-to-resources-iam#grant_access_to_a_dataset):
+
+![Sharing dataset](../../images/docs/batch-exports/bigquery/dataset-sharing.png)
+![Add principal](../../images/docs/batch-exports/bigquery/dataset-add-principal.png)
+
+Use the Sharing and Add Principal buttons to share your dataset with your new Service Account created in step 1.
+
+Grant either the `BigQuery Data Owner` role, or your custom role created in step 4 to the Service Account. In the screenshot below, we have used a custom role named `Testing PostHog BatchExports`.
+
+![Sharing dataset](../../images/docs/batch-exports/bigquery/dataset-grant-access.png)
+
+8. All done! After completing these steps you can create a BigQuery batch export and your data will start flowing from PostHog to BigQuery.
+
+## Event schema
+
+This is the schema of all the fields that are exported to BigQuery.
+
+| Field                 | Type        | Description                                                                      |
+|-----------------------|-------------|----------------------------------------------------------------------------------|
+| uuid                  | `STRING`    | The unique ID of the event within PostHog                                        |
+| event                 | `STRING`    | The name of the event that was sent                                              |
+| properties            | `STRING`    | A JSON object with all the properties sent along with an event                   |
+| elements              | `STRING`    | A string of elements surrounding an [autocaptured](/docs/data/autocapture) event |
+| set                   | `STRING`    | A JSON object with any person properties sent with the `$set` field              |
+| set_once              | `STRING`    | A JSON object with any person properties sent with the `$set_once` field         |
+| distinct_id           | `STRING`    | The `distinct_id` of the user who sent the event                                 |
+| team_id               | `STRING`    | The `team_id` for the event                                                      |
+| ip                    | `STRING`    | The IP address that was sent with the event                                      |
+| site_url              | `STRING`    | The $current_url property of the event                                           |
```

Comment:
Can we drop this everywhere? Context for what it is https://posthog.slack.com/archives/C0374DA782U/p1687955284804389 it's essentially only ever used for when webhooks link to the person etc, so it likely shouldn't be in batch exports anywhere

Reply:
We can. However, the docs reflect how the system looks right now. Once we drop the field, we can remove this from here.

Reply:
any reason not to drop the field now?

Reply:
The code is not here.

Reply:
In other words, I don't know what this question is doing in a PR about docs. Happy to review a PR to remove this over at https://github.com/PostHog/posthog.

Reply:
The fair thing to ask in _this PR_ is to warn users the field will be deprecated, which I have done in my latest commit.

Reply:
I don't impose restrictions like that on my PR comments & I prefer if others don't either. 
I either didn't notice or review your PR that added site_url, so I commented about it the first time I saw/noticed it - referring to the best actual solution.